gatherer/src/main/java/de/zalando/pgobserver/gatherer/GathererApp.java

```diff
@@ -76,7 +76,7 @@ public static void main(final String[] args) {
 
         try {
             LOG.info("Starting restlet server");
-            new Server(Protocol.HTTP, 8182, GathererApp.class).start();
+            new Server(Protocol.HTTP, Integer.parseInt(getEnv("PORT0", "8182")), GathererApp.class).start();
```

Comment:
Could we have a more descriptive name for that environment variable? Maybe "HTTP_PORT"?

Reply:
Variable `PORT0` is automatically set by Marathon orchestrator, thus allowing seamless integration.